front-end/src/components/EventCard.js

```diff
@@ -3,18 +3,19 @@ import {Card, CardHeader, CardText, CardMedia} from 'material-ui/Card';
 import TweetEmbed from 'react-tweet-embed';
 import ReactPlayer from 'react-player';
 
+import { timer } from '../actions/index'
+import { connect } from 'react-redux';
```

Comment:
Disconnect this from redux, it has no connection to the state nor the dispatcher.